storage/Utils/FileUtils.cc

```diff
@@ -37,6 +43,23 @@
 namespace storage
 {
 
+    bool create_file_if_not_exists(const string& filename, const int permissions)
+    {
+	int file_descriptor = open(filename.c_str(), O_RDWR | O_APPEND | O_CREAT, permissions);
+
+	if (file_descriptor < 0)
+	{
+	    y2err("open(" << filename << ") failed");
```

Comment:
Please also log `errno` and `strerror(errno)` so we have a chance to find out **why** it failed.